From dcc77fe1671a99b98700dbec5c9573a5407a1601 Mon Sep 17 00:00:00 2001
From: Till Kamppeter <till.kamppeter@gmail.com>
Date: Fri, 22 Jul 2016 09:33:01 +0200
Subject: Make the menu of the hp-systray applet appearing when the applet is
 converted to an indicator applet by sni-qt and put into the notification area
 of KDE

LP: #857929
Patch-Name: hp-systray-make-menu-appear-in-sni-qt-indicator-with-kde.patch
---
 ui5/systemtray.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/ui5/systemtray.py b/ui5/systemtray.py
index 8888dec..ec3574f 100644
--- a/ui5/systemtray.py
+++ b/ui5/systemtray.py
@@ -576,6 +576,9 @@ class SystemTrayApp(QApplication):
 
         elif reason == QSystemTrayIcon.Trigger:
             #print "single click"
+            self.updateMenu()
+            if not self.menu is None:
+                self.menu.popup(QCursor.pos())
             pass
 
         elif reason == QSystemTrayIcon.MiddleClick:
